### PR TITLE
- Added new release of Geonetwork

### PR DESCRIPTION
**Pull request by doublebyte1**
- Updated docker repository with new images for the GN latest release

### CODE REVIEW

File: library/geonetwork [hunk 1]
```diff
@@ -11,10 +11,18 @@ Tags: 3.0.5-postgres, 3.0-postgres
 GitCommit: e5cf5da76f557481c35b4a4d3e3cac77768a1302
 Directory: 3.0.5/postgres
 
-Tags: 3.2.0, 3.2, latest
+Tags: 3.2.0
 GitCommit: c2af30125f631f6357d3af3837ec4b80f04b5917
 Directory: 3.2.0
```
Comment (tianon) [inline library/geonetwork]: Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (doublebyte1) [inline library/geonetwork]: @tianon I agree. Please check my latest commit, thanks.

Comment (doublebyte1) [inline library/geonetwork]: @tianon I agree. Please check my latest commit, thanks.

**Comment by yosifkit** [comment]
```diff
diff --git a/geonetwork_latest/Dockerfile b/geonetwork_latest/Dockerfile
index 3a94850..bbbc2d1 100644
--- a/geonetwork_latest/Dockerfile
+++ b/geonetwork_latest/Dockerfile
@@ -2,11 +2,11 @@ FROM tomcat:8.0-jre8
 
 ENV GN_FILE geonetwork.war
 ENV DATA_DIR=$CATALINA_HOME/webapps/geonetwork/WEB-INF/data
-ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -Xmx512M -Xss2M -XX:MaxPermSize=512m -XX:+UseConcMarkSweepGC"
+ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:PermSize=512m -XX:MaxPermSize=1024m -XX:+UseConcMarkSweepGC"
 
 #Environment variables
-ENV GN_VERSION 3.2.0
-ENV GN_DOWNLOAD_MD5 87a84ffb3fbbd662d595c08e1a7fdff2
+ENV GN_VERSION 3.2.1
+ENV GN_DOWNLOAD_MD5 54140c3e4badc87651bd656f13f860f2
 
 WORKDIR $CATALINA_HOME/webapps
 
diff --git a/geonetwork_postgres/Dockerfile b/geonetwork_postgres/Dockerfile
index f756469..e25c930 100644
--- a/geonetwork_postgres/Dockerfile
+++ b/geonetwork_postgres/Dockerfile
@@ -1,4 +1,4 @@
-FROM geonetwork:3.2.0
+FROM geonetwork:3.2.1
 
 RUN apt-get update && apt-get install -y postgresql-client && \
     rm -rf /var/lib/apt/lists/*
```

**Comment by yosifkit** [comment]
Build test of #2685; e42927c (`geonetwork`):

```console

$ bashbrew build geonetwork:3.0.5
Building bashbrew/cache:3d243e71c3d7fa8284f5e25cfe38b84a268dfb54c06c17e1d016ec5cf0fdce4b (geonetwork:3.0.5)
Tagging geonetwork:3.0.5
Tagging geonetwork:3.0

$ test/run.sh geonetwork:3.0.5
testing geonetwork:3.0.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.0.5-postgres
Building bashbrew/cache:c5922f325d09a5d196818d71e2198460896ee414c24a113ca0662ee98af7f6b7 (geonetwork:3.0.5-postgres)
Tagging geonetwork:3.0.5-postgres
Tagging geonetwork:3.0-postgres

$ test/run.sh geonetwork:3.0.5-postgres
testing geonetwork:3.0.5-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.2.1
Building bashbrew/cache:1d99af249398d08262b28cd0ce584ab18956d4197cdfa4ed487abd3d479c65f0 (geonetwork:3.2.1)
Tagging geonetwork:3.2.1
Tagging geonetwork:3.2
Tagging geonetwork:latest

$ test/run.sh geonetwork:3.2.1
testing geonetwork:3.2.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.2.1-postgres
Building bashbrew/cache:39ab1f8d9afbf885ee8b2da01527b7dd884e64b17c507b780316072cba05cbc1 (geonetwork:3.2.1-postgres)
Tagging geonetwork:3.2.1-postgres
Tagging geonetwork:3.2-postgres
Tagging geonetwork:postgres

$ test/run.sh geonetwork:3.2.1-postgres
testing geonetwork:3.2.1-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**